--- prosody-0.11.5/prosody.cfg.lua.dist.orig	2020-01-19 16:50:32.000000000 +0100
+++ prosody-0.11.5/prosody.cfg.lua.dist	2020-04-04 11:24:01.668047272 +0200
@@ -153,11 +153,12 @@
 -- Logging configuration
 -- For advanced logging see https://prosody.im/doc/logging
 log = {
-	info = "prosody.log"; -- Change 'info' to 'debug' for verbose logging
-	error = "prosody.err";
-	-- "*syslog"; -- Uncomment this for logging to syslog
+	-- info = "prosody.log"; -- Change 'info' to 'debug' for verbose logging
+	-- error = "prosody.err";
+	"*syslog"; -- Uncomment this for logging to syslog
 	-- "*console"; -- Log to the console, useful for debugging with daemonize=false
 }
+pidfile = "/run/prosody/prosody.pid";
 
 -- Uncomment to enable statistics
 -- For more info see https://prosody.im/doc/statistics
